tests/foreman/sys/test_sat_clone.py

```diff
@@ -0,0 +1,129 @@
+# -*- encoding: utf-8 -*-
+"""Test class for ``satellite clone``
+
+:Requirement: SatClone
+
+:CaseAutomation: notautomated
+
+:CaseLevel: System
+
+:CaseComponent: Clone
+
+:TestType: functional
+
+:CaseImportance: High
+
+:Upstream: No
+
+:Setup:
+    1. Upload manifest
+    2. Enable and sync RH products
+    3. Create content views - publish, promote"
+    4. Create Activation keys
+    5. Create custom products, custom repos ((both yum and puppet), sync,
+    create content views - publish, promote
+    6. Provision and Register content hosts
+    7. Create an other test org
+    8. Create a new user
+    9. Create a Discovery rule
+
+"""
+from robottelo.decorators import (
+        destructive,
+        stubbed,
+)
+from robottelo.test import TestCase
+
+
+@destructive
+class SatCloneTestCase(TestCase):
+    """Implements ``Satellite Clone`` tests"""
+
+    @destructive
+    @stubbed()
+    def test_clone_without_rename(self):
+        """Clone the satellite to other box
+
+        :id: 8dc01d48-5930-43a1-8a34-2fc150482541
+
+        :Steps:
+            1. Refresh the manifest
+            Enable & sync new RH repo
```

Comment:
`rst` formatting error here, or you increase indentation or you add more listing numbers.